test/parser/expression/__snapshots__/AssociativeArrayLiterals.test.js.snap

```diff
@@ -0,0 +1,239 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
How do these snapshots work?  Are they part of the test process?  I don't see how they're referenced.

Reply:
They're provided by the Jest unit testing framework, and are used to check for unexpected changes in a JavaScript object.  In our case, they're used mostly to break tests when the parser's output changes unexpectedly.  https://goo.gl/fbAQLP (the link in that comment) resolves to https://jestjs.io/docs/snapshot-testing.html and has more information.